desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -12,6 +14,8 @@
 
 #define COMBO_CHANGED 0
 #define TEXT_EDITED 1
+#define CSS_SET_HEADING_BLUE "QLabel { color: #0000c0;} "
+#define CSS_SET_SMALL_LABEL "QLabel { color: #0000c0; font-size: 7pt;}"
```

Comment:
I always worry when we hard-code sizes. What about visually impaired people who run their screen with a 30pt base font - this will be ridiculously small for them. I think it would be better to use `font-size: small` or `font-size: x-small`

Reply:
Qt only does pixel or point sizes, as far as I can see in the documentation. It does not implement em sizes or names sizes such as small. Even when implementing a dedicated Font class, the size attributes are only in pixels or points. I have checked throughout the Subsurface UI and I cannot find another case where smaller font is used. Any ideas?

Reply:
I did this by getting the height of a label in pixels. Then use a pixel point size that is a fraction of that of the height of a label.